metasrv/src/bin/metasrv.rs

```diff
@@ -75,14 +75,16 @@ async fn main(_global_tracker: Arc<RuntimeTracker>) -> common_exception::Result<
         let mut srv = GrpcServer::create(conf.clone(), meta_node.clone());
         tracing::info!(
             "Databend meta server listening on {}",
-            conf.grpc_api_address
+            conf.grpc_api_address.clone()
         );
         srv.start().await.expect("Databend meta service error");
         stop_handler.push(Box::new(srv));
     }
 
     // join raft cluster after all service started
-    meta_node.join_cluster(&conf.raft_config).await?;
+    meta_node
+        .join_cluster(&conf.raft_config, conf.grpc_api_address)
```

Comment:
I'm confused about the `raft_config`.

Will we use `raft_config` outside `metasrv`? How about merging them into `MetaConfig` directly?

Reply:
`raft_config` is all about raft related config, `MetaConfig` includes other configs that nothing to do with raft. So I think I think it's better to separate.

Reply:
Right...It's a bit dirty:

RaftConfig is also used by `common/meta/embedded`, which is an embedded meta server for testing purposes.